sklearn/preprocessing/data.py

```diff
@@ -397,6 +396,256 @@ def inverse_transform(self, X, copy=None):
         return X
 
 
+class RobustScaler(BaseEstimator, TransformerMixin):
+    """Standardize features by removing the median and scaling to IQR.
```

Comment:
You use the word "standardize" here and also in the docstring of the function. Maybe not good as discussed below. The other scaler is called "StandardScaler" so maybe just say "Scale features"? I don't know, and that is anyhow maybe to much nitpicking ;)